common/.gitignore

```diff
@@ -0,0 +1,14 @@
+# Compiled class file
```

Comment:
Can we perhaps use a gitignore generator 
EG : https://www.toptal.com/developers/gitignore

Reply:
hey generated file is too big IMO
https://www.toptal.com/developers/gitignore/api/intellij,kotlin